sdk/healthinsights/health-insights-cancerprofiling-rest/review/health-insights-cancerprofiling.api.md

```diff
@@ -0,0 +1,270 @@
+## API Report File for "@azure-rest/health-insights-cancerprofiling"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { Client } from '@azure-rest/core-client';
+import { ClientOptions } from '@azure-rest/core-client';
+import { CreateHttpPollerOptions } from '@azure/core-lro';
+import { ErrorResponse } from '@azure-rest/core-client';
+import { HttpResponse } from '@azure-rest/core-client';
+import { KeyCredential } from '@azure/core-auth';
+import { OperationState } from '@azure/core-lro';
+import { RawHttpHeaders } from '@azure/core-rest-pipeline';
+import { RawHttpHeadersInput } from '@azure/core-rest-pipeline';
+import { RequestParameters } from '@azure-rest/core-client';
+import { SimplePollerLike } from '@azure/core-lro';
+import { StreamableMethod } from '@azure-rest/core-client';
+
+// @public (undocumented)
+export type CancerProfilingRestClient = Client & {
+    path: Routes;
+};
+
+// @public
+export interface ClinicalCodedElement {
+    code: string;
+    name?: string;
+    system: string;
+    value?: string;
+}
+
+// @public
+export interface ClinicalCodedElementOutput {
+    code: string;
+    name?: string;
+    system: string;
+    value?: string;
+}
+
+// @public
+export interface ClinicalNoteEvidenceOutput {
+    id: string;
+    length: number;
+    offset: number;
+    text?: string;
+}
+
+// @public
+function createClient(endpoint: string, credentials: KeyCredential, options?: ClientOptions): CancerProfilingRestClient;
+export default createClient;
+
+// @public (undocumented)
+export interface CreateJob {
+    post(options?: CreateJobParameters): StreamableMethod<CreateJob200Response | CreateJob202Response | CreateJobDefaultResponse>;
+}
+
+// @public
+export interface CreateJob200Response extends HttpResponse {
+    // (undocumented)
+    body: OncoPhenotypeResultOutput;
+    // (undocumented)
+    status: "200";
+}
+
+// @public (undocumented)
+export interface CreateJob202Headers {
+    "operation-location": string;
+    "repeatability-result"?: "accepted" | "rejected";
```

Comment:
Just out of curious, what should client side do, if it gets a "repeatability-result" "accepted" or "rejected" ? Is it possible to provide example on how to handle it ?

Reply:
You can read here about repeatability request - https://docs.oasis-open.org/odata/repeatable-requests/v1.0/cs01/repeatable-requests-v1.0-cs01.html
And good explanation here - https://github.com/Azure/autorest.csharp/issues/1950
About the example - will be added when we go to GA

Reply:
I understand the background, but I am curious why need to wait for GA, and have other language SDKs supported this ?  Just want to be clear, in RLC you have the full control of any headers, so if the backend is ready, there's no need to wait in RLC.

Reply:
Its actually a matter of prioritization. For the private preview we are satisfied with it.